tests/Persistence/PageContentConfigLookupTest.php

```diff
@@ -11,6 +11,7 @@
 
 /**
  * @covers \ProfessionalWiki\WikibaseFacetedSearch\Persistence\PageContentConfigLookup
+ * @uses \ProfessionalWiki\WikibaseFacetedSearch\Persistence\PageContentFetcher
```

Comment:
It is common to split code into several classes but only have tests for the higher level one. In such cases it's good to include covers tags for the lower level ones in said test.